Makefile

```diff
@@ -62,22 +69,22 @@ dscr.a51: dscr.a51.in
           -e"s/\$$H9/$${H:9:1}/" | \
 	 sed \
 	  -e"s/        .db        '~, 0//" \
-	 > dscr.a51
+	 > $@
 
 #%.iic : %.hex
 #	./hex2bix -ir -f 0xC2 -m 0xF000 -c 0x1 -o $@ $<
 %.bix: %.hex
 	objcopy -I ihex -O binary $< $@
 
-usbjtag.hex: vectors.rel usbjtag.rel dscr.rel eeprom.rel ${HARDWARE}.rel startup.rel ${LIBDIR}/${LIB}
+temp-%.hex: vectors.rel usbjtag.rel dscr_%.rel eeprom.rel hw_%.rel startup.rel ${LIBDIR}/${LIB}
```

Comment:
Maybe these are better named "usbjtag-%.ihx" (a common suffix for Intel Hex Format)?